src/BackupMetadata.cs

```diff
@@ -89,15 +89,29 @@ public object Clone()
     }
   }
 
-  public class StripedBackupSet
+  public class StripedBackupSet : BackupMetadata
   {
     public IEnumerable<BackupMetadata> StripedBackups { get; private set; }
 
     private StripedBackupSet(IEnumerable<BackupMetadata> stripedBackups)
     {
       StripedBackups = stripedBackups;
+      SetMetadata(StripedBackups.First());
     }
 
+    private void SetMetadata(BackupMetadata firstStripe)
+    {
+     CheckpointLsn = firstStripe.CheckpointLsn;
+     DatabaseBackupLsn = firstStripe.DatabaseBackupLsn;
+     DatabaseName = firstStripe.DatabaseName;
+     FirstLsn = firstStripe.FirstLsn;
+     LastLsn = firstStripe.LastLsn;
+     PhysicalDeviceName = null;

```

Comment:
I'm on the fence about setting `null` for `PhysicalDeviceName`.

Have debated with myself for a good 5 mins and come to the following conclusion 😅:
"the `null` here signifies that there isn't just **one** physical device name" 

This sounds ok to me, not sure if there is a better way to do this